release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -264,6 +264,11 @@ In {product-title} {product-version}, {lvms} is supported in dual-stack for IPv4
 [id="ocp-4-13-osdk"]
 === Operator development
 
+[id="ocp-4-13-nto"]
+==== Node Tuning Operator
+
+The Node Tuning Operator (NTO) is now optional during installation and can be disabled after installation if desired. For more information, see xref:../installing/cluster-capabilities.adoc#about-node-tuning-operator_cluster-capabilities[Node Tuning capability].
```

Comment:
This isn't quite right. NTO cannot be disabled after installation. If it is disabled during installation, it can be enabled later. If it is enabled during installation, it cannot be disabled later.

```suggestion
The Node Tuning Operator (NTO) can now be enabled/disabled using the `NodeTuning` cluster capability. If disabled at cluster install, it can be re-enabled later. For more information, see xref:../installing/cluster-capabilities.adoc#about-node-tuning-operator_cluster-capabilities[Node Tuning capability].
```

Reply:
I knew I was going to get it backwards! I updated the pull request. Thank you!